### ISSUE TITLE
NPE OverridePrototypeAction

### ISSUE THREAD

**Issue by astrelsky** [NationalSecurityAgency/ghidra#2177]
**Attachments**
<details><summary>stacktrace</summary>

```
Uncaught Exception! 
NullPointerException - Cannot invoke "ghidra.program.model.pcode.PcodeOp.getOpcode()" because "op" is null
java.lang.NullPointerException: Cannot invoke "ghidra.program.model.pcode.PcodeOp.getOpcode()" because "op" is null
	at ghidra.app.plugin.core.decompile.actions.OverridePrototypeAction.getCallOp(OverridePrototypeAction.java:115)
	at ghidra.app.plugin.core.decompile.actions.OverridePrototypeAction.isEnabledForDecompilerContext(OverridePrototypeAction.java:218)
	at ghidra.app.plugin.core.decompile.actions.AbstractDecompilerAction.lambda$isEnabledForContext$0(AbstractDecompilerAction.java:58)
	at ghidra.app.plugin.core.decompile.DecompilerActionContext.checkActionEnablement(DecompilerActionContext.java:147)
	at ghidra.app.plugin.core.decompile.actions.AbstractDecompilerAction.isEnabledForContext(AbstractDecompilerAction.java:57)
	at docking.ComponentPlaceholder.contextChanged(ComponentPlaceholder.java:501)
	at docking.DockingWindowManager.contextChanged(DockingWindowManager.java:2113)
	at docking.AbstractDockingTool.contextChanged(AbstractDockingTool.java:208)
	at ghidra.framework.plugintool.PluginTool.contextChanged(PluginTool.java:1485)
	at ghidra.app.plugin.core.decompile.DecompilerProvider.contextChanged(DecompilerProvider.java:684)
	at ghidra.app.plugin.core.decompile.DecompilerProvider.locationChanged(DecompilerProvider.java:524)
	at ghidra.app.decompiler.component.DecompilerController.locationChanged(DecompilerController.java:254)
	at ghidra.app.decompiler.component.DecompilerPanel.fieldLocationChanged(DecompilerPanel.java:769)
	at docking.widgets.fieldpanel.FieldPanel$CursorHandler.notifyCursorChanged(FieldPanel.java:2125)
	at docking.widgets.fieldpanel.FieldPanel$CursorHandler.setCursorPos(FieldPanel.java:1920)
	at docking.widgets.fieldpanel.FieldPanel$FieldPanelMouseAdapter.mousePressed(FieldPanel.java:1307)
	at java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6633)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6401)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5012)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4844)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4919)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4545)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4489)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2764)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4844)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

---------------------------------------------------
Build Date: 2020-Aug-03 1814 EDT
Ghidra Version: 9.2
Java Home: C:\Program Files\Java\jdk-14.0.2
JVM Version: Oracle Corporation 14.0.2
OS: Windows 10 10.0 amd64
Workstation: 
```
</details>

This is occurring when clicking on a `goto LAB_*`.

**Comment by dragonmacher** [comment]
I must have broken this in the last change.

**Comment by dragonmacher** [comment]
Fixed by 61e7417
